schematic/manifest/generator.py

```diff
@@ -1677,16 +1682,25 @@ def create_manifests(
                     "Please check your submission and try again."
                 )
 
-        data_model_parser = DataModelParser(path_to_data_model=path_to_data_model)
+        if not graph_data_model:
```

Comment:
You might want this to be `if graph_data_model is not None` I'm not sure how `nx.MultiDiGraph` handles thruthyness.